deploy/docs/opentelemetry_collector.md

```diff
@@ -60,148 +77,71 @@ sumologic:
       provider: otelcol
 ```
 
-As we are providing drop-in replacement, most of the configuration from
-[`values.yaml`][values] should work
-the same way for Opentelemetry Collector like for Fluentd.
+If you haven't modified the FluentD or Fluent-Bit configuration, this should be a drop-in replacement with no
+further changes required.
 
 ### Logs Configuration
 
-All Opentelemetry Collector configuration for logs is located in
-[`values.yaml`][values] as `metadata.logs.config`.
+High level Opentelemetry Collector configuration for logs is located in
+[`values.yaml`][values] under the `sumologic.logs`.
 
-If you want to modify it, please see [Sumologic Opentelemetry Collector configuration][configuration]
-for more information.
+Configuration specific to the log collector Daemonset can be found under the `otellogs` key.
+
+Finally, configuration specific to the metadata enrichment Statefulset can be found under the `metadata.logs` key.
+
+In both of the aforementioned cases, the raw configuration can be overridden - this is done respectively by using
+the `otellogs.config.override` and `metadata.logs.config` sections. Only use these if your use case isn't covered
+by the high-level settings. See [Sumologic Opentelemetry Collector configuration][configuration]
+for more information
 
 [configuration]: https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Configuration.md
 [values]: ../helm/sumologic/values.yaml
 
-## Scraping Containers Logs
+### Multiline log parsing
+
+Multiline log parsing for Opentelemetry Collector can be configured using the `sumologic.logs.multiline` section in [values.yaml][values].
```

Comment:
```suggestion
Multiline log parsing for OpenTelemetry Collector can be configured using the `sumologic.logs.multiline` section in [values.yaml][values].
```

Reply:
This is actually wrong elsewhere in the document as well, I can fix it while I'm at it.